src/EventManagement.Web/Controllers/Api/EmailController.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using losol.EventManagement.Services;
+using losol.EventManagement.Web.Services;
+using losol.EventManagement.ViewModels;
+
+namespace losol.EventManagement.Web.Controllers.Api
+{
+    [Authorize]
```

Comment:
Does this authorize all users to use the email api? That might be okay, however on the SMS api I think we should authorize roles ie admin, superadmin.

Reply:
@losviko I've addressed this in PR #60